components/Comment/CommentAnonymously.tsx

```diff
@@ -0,0 +1,29 @@
+import { ReactElement } from "react";
+import Switch from "@mui/material/Switch";
+import { css, StyleSheet } from "aphrodite";
+
+type Props = {
+  id: string;
+  anonymous: boolean;
+  onToggle: () => void;
+};
+
+const AnonymousToggle = ({ id, anonymous, onToggle }: Props): ReactElement => {
+  return (
+    <div className={css(styles.toggleContainer)}>
+      toggle anonymity
```

Comment:
typing these html strings as interpolated string makes it easy to code-search for other developers. Also, it keeps the trailing / leading white spaces as typed rather than relying on the browser to interpret these themselves ( usually browsers remove them ) 
```suggestion
      {"toggle anonymity"}
```

Reply:
refactored